build/ci/community/wait_demo_pr_checks.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash -xe
+set +o pipefail
+
+gh_pr_checks_command (){
+  community_operators_branch=$1
+  gh pr checks $community_operators_branch --repo $forked_community_operators_repository
+}
+
+wait_fot_checks_to_start(){
+  community_operators_branch=$1
+  forked_community_operators_repository=$2
```

Comment:
is this necessary for `gh_pr_checks_command`?

Reply:
if both are passed, then I think their name inside the `gh_pr_checks_command` function should be more general, e.g. `branch` and `repository`